### PR TITLE
[tcl/en] Rework some examples.

### PR DESCRIPTION
**Pull request by pooryorick**
Reworked examples, reworded some descriptions, cleaned up comment layout.

### CODE REVIEW

File: tcl.html.markdown [hunk 1]
```diff
@@ -30,9 +32,9 @@ Various features of Tcl stand out:
 * A threading model recognized as robust and easy to use
 
 
-If Lisp is a list processor, then Tcl is a string processor.  All values are
-strings.  A list is a string format.  A procedure definition is a string
-format.  To achieve performance, Tcl internally caches structured
+Tcl is a Lisp built on the foundation of string processing.  All values are
```
Comment (adambard) [inline tcl.html.markdown]: Upon googling this is a borderline defensible statement, but if you leave "Tcl is a Lisp" in there it's just going to be reverted by some uptight lisper.

File: tcl.html.markdown [hunk 2]
```diff
@@ -50,6 +52,10 @@ syntax of other languages, allows it to fade into the background of whatever
 domain-specific functionality a project needs. It's syntax, which is even
 lighter that that of Lisp, just gets out of the way.
 
+There is a document floating around the web titled "Why you should not use
+Tcl".  Ignore that.  It's woefully out-of-date, and even when it was published
+was mostly FUD.
+
```
Comment (Lekensteyn) [inline tcl.html.markdown]: Most of the formatting/typo changes seem OK, but isn't this statement a bit unnecessary for an introduction?

**Comment by vendethiel** [comment]
can we avoid the "this is lisp" stuff? thanks!

**Comment by vendethiel** [comment]
Going to close this one for now :). Feel free to ping us here if you find some time working on it again! Thanks.

**Pull request closed without merging**